src/rosdep2/lookup.py

```diff
@@ -171,7 +174,7 @@ def __str__(self):
 \trosdep key : %s
 \tOS name    : %s
 \tOS version : %s
-\tData: %s""" % (self.args[0], self.rosdep_key, self.os_name, self.os_version, pretty_data)
+\tData:\n%s""" % (self.args[0], self.rosdep_key, self.os_name, self.os_version, pretty_data)
```

Comment:
What do you think about doing something like this as well?

```suggestion
\tData:\n%s""" % (self.args[0], self.rosdep_key, self.os_name, self.os_version, pretty_data.replace('\n', '\n\t\t'))
```

Reply:
Sure :+1: